analyzers/tests/SonarAnalyzer.UnitTest/Rules/TestMethodShouldContainAssertionTest.cs

```diff
@@ -38,6 +38,12 @@ private static class FluentAssertionVersions
             public const string Ver5 = "5.9.0";
         }
 
+        private static class NFluentVersions
```

Comment:
where are you using these? you're only using `Ver1` for legacy, and in the rest you are using the default `NuGetLatestVersion`, right?

Do you need this static class at all, then?